zebra/zebra_mpls.c

```diff
@@ -1527,7 +1530,8 @@ static json_object *nhlfe_json(zebra_nhlfe_t *nhlfe)
 					       ifindex2ifname(nexthop->ifindex,
 							      nexthop->vrf_id));
 		break;
-	default:
+	case NEXTHOP_TYPE_BLACKHOLE:
+	case NEXTHOP_TYPE_IFINDEX:
```

Comment:
TYPE_IFINDEX does appear to be supported - it should print something here?

Reply:
same for blackhole.   But I was not sure what to do.  I believe but do not know that you shouldn't be able to create either of these types here.

Reply:
hmm - there are some comments about supporting TYPE_IFINDEX, so that may be a possibility even if TYPE_BLACKHOLE is not

Reply:
yeah I think that there are bugs in here.  I don't know how to fix these here though.  In any event I'm not doing anything here other than moving the default case statement to individual.  

In the long run this is exactly why I hate allowing default: as a case statement for a well defined enumeration.  We miss code that should be changed when we add stuff.

Reply:
ok, fair enough. I'll make a 'todo' to come back around and at least fill out the TYPE_IFINDEX clauses for the cli output.